containers/Prices.ts

```diff
@@ -130,6 +137,15 @@ function usePrices() {
       vs_currencies: ["usd"],
```

Comment:
I noticed `okexchain` above throws, needs to be `oec-token` and  `wokt: response["oec-token"].usd,` below